Test fails for sendfile with EINVAL

```python
==================================================================================================== FAILURES =====================================================================================================
_______________________________________________________________________________________________ test_sendfile_file ________________________________________________________________________________________________

tmpdir = local('/tmp/pytest-of-nwani/pytest-6/test_sendfile_file0')

    @pytest.mark.asyncio
    def test_sendfile_file(tmpdir):
        """Test the sendfile functionality, file-to-file."""
        filename = join(dirname(__file__), 'resources', 'test_file1.txt')
        tmp_filename = tmpdir.join('tmp.bin')

        with open(filename) as f:
            contents = f.read()

        input_file = yield from aiofiles.open(filename)
        output_file = yield from aiofiles.open(str(tmp_filename), mode='w+')

        size = (yield from aiofiles.os.stat(filename)).st_size

        input_fd = input_file.fileno()
        output_fd = output_file.fileno()

>       yield from aiofiles.os.sendfile(output_fd, input_fd, 0, size)

tests/test_os.py:35:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../_t_env/lib/python3.6/asyncio/coroutines.py:213: in coro
    res = yield from res
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <concurrent.futures.thread._WorkItem object at 0x2af170783ef0>

    def run(self):
        if not self.future.set_running_or_notify_cancel():
            return

        try:
>           result = self.fn(*self.args, **self.kwargs)
E           OSError: [Errno 22] Invalid argument

../_t_env/lib/python3.6/concurrent/futures/thread.py:55: OSError
====================================================================================== 1 failed, 229 passed in 6.33 seconds =======================================================================================
```

Strace to the rescue:

```bash
669   sendfile(19, 18, [0], 10)         = -1 EINVAL (Invalid argument)
```

--- a/tests/test_os.py
+++ b/tests/test_os.py
@@ -3,6 +3,7 @@ import aiofiles.os
 import asyncio
 from os.path import join, dirname
 import pytest
+import platform
 
 
 @pytest.mark.asyncio
@@ -15,6 +16,8 @@ def test_stat():
     assert stat_res.st_size == 10
 
 
+@pytest.mark.skipif('2.4' < platform.release() < '2.6.33',
+                    reason = "sendfile() syscall doesn't allow file->file")
 @pytest.mark.asyncio
 def test_sendfile_file(tmpdir):
     """Test the sendfile functionality, file-to-file."""

